kano_profile/tracker/__init__.py

```diff
@@ -1,227 +1,52 @@
-#!/usr/bin/env python
-
-# tracker.py
+#
+# __init__.py
 #
 # Copyright (C) 2014 - 2017 Kano Computing Ltd.
 # License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
 #
+# Kano-tracker module
+#
+# A small module for tracking various metrics the users do in Kano OS
+#
 
-"""
-Kano-tracker module
 
-A small module for tracking various metrics the users do in Kano OS
-"""
+__author__ = 'Kano Computing Ltd.'
+__email__ = 'dev@kano.me'
+
 
 import time
 import atexit
 import datetime
-import fcntl
 import json
 import os
-import hashlib
 import subprocess
 import shlex
 
 from uuid import uuid1, uuid5
 
-from kano.utils import get_program_name, is_number, read_file_contents, \
-    get_cpu_id, chown_path, ensure_dir
+from kano.utils.file_operations import read_file_contents, chown_path, \
```

Comment:
I guess we have to have all of this in `__init__.py`because of backwards compat?

Reply:
Yup